src/backend/marsha/core/tests/test_views_lti_select.py

```diff
@@ -109,6 +111,76 @@ def test_views_lti_select(self):
         lti_parameters.update({"lti_message_type": "ContentItemSelection"})
         self.assertEqual(jwt_token.get("lti_select_form_data"), lti_parameters)
 
+    def test_views_lti_select_behind_tls_termination_proxy(self):
+        """Validate the context passed to the frontend app for a LTI Content selection."""
+        lti_consumer_parameters = {
+            "roles": random.choice(["instructor", "administrator"]),
+            "content_item_return_url": "https://lti-consumer.site/lti",
+            "context_id": "sent_lti_context_id",
+        }
+        lti_parameters, passport = generate_passport_and_signed_lti_parameters(
+            url="https://testserver/lti/select/",
+            lti_parameters=lti_consumer_parameters,
+        )
+
+        resolutions = [144]
+        playlist = PlaylistFactory(
+            lti_id=lti_parameters.get("context_id"),
+            consumer_site=passport.consumer_site,
+        )
+        video = VideoFactory(
+            playlist=playlist,
+            uploaded_on=timezone.now(),
+            resolutions=resolutions,
+        )
+        document = DocumentFactory(
+            playlist=playlist,
+            uploaded_on=timezone.now(),
+        )
+
+        response = self.client.post(
+            "/lti/select/",
+            lti_parameters,
+            HTTP_REFERER="https://testserver",
```

Comment:
should not be `http://testserver` ? In this test, with or with the `HTTP_X_FORWARDED_PROTO` you will have the same result I think.